software/base/src/main/java/brooklyn/entity/brooklynnode/BrooklynEntityMirror.java

```diff
@@ -54,6 +56,9 @@
         "Frequency to poll for client sensors", Duration.FIVE_SECONDS);
     
     public static final AttributeSensor<String> MIRROR_STATUS = Sensors.newStringSensor("brooklyn.mirror.monitoring_status");
+    @SuppressWarnings("rawtypes")
+    public static final AttributeSensor<Map> MIRROR_SUMMARY = Sensors.newSensor(Map.class, "brooklyn.mirror.summary");
+    public static final AttributeSensor<String> MIRROR_CATALOG_ITEM_ID = Sensors.newStringSensor("brooklyn.mirror.catalog_item_id");
```

Comment:
add description, noting that this is the catalog item id in the remote brooklyn of the mirrored entity, which could be different to the local catalog item id